e2e/metrics/input/cpustress.nomad

```diff
@@ -10,33 +10,22 @@ job "cpustress" {
   group "cpustress" {
     count = 1
 
-    restart {
```

Comment:
Note to reviewers: I cut out some of the cruft from these inputs to reduce noise in understanding what we're testing. A few got reduced in resources so they'd fit on the test client nodes, and I dropped the `redis` workload entirely because it wasn't telling us anything the others didn't.